test/images/webhook/main.go

```diff
@@ -164,18 +164,18 @@ func serve(w http.ResponseWriter, r *http.Request, admit admitFunc) {
 		return
 	}
 
-	var reviewStatus *v1alpha1.AdmissionReviewStatus
+	var reviewResponse *v1alpha1.AdmissionResponse
 	ar := v1alpha1.AdmissionReview{}
 	deserializer := codecs.UniversalDeserializer()
 	if _, _, err := deserializer.Decode(body, nil, &ar); err != nil {
 		glog.Error(err)
-		reviewStatus = toAdmissionReviewStatus(err)
+		reviewResponse = toAdmissionResponse(err)
 	} else {
-		reviewStatus = admit(ar)
+		reviewResponse = admit(ar)
 	}
 
-	if reviewStatus != nil {
-		ar.Status = *reviewStatus
+	if reviewResponse != nil {
+		ar.Response = reviewResponse
```

Comment:
Set reviewResponse.UID?

Reply:
Fixed it slightly differently, please take a look.